src/inc/msquic.h

```diff
@@ -295,6 +295,18 @@ typedef struct QUIC_LISTENER_STATISTICS {
     } Binding;
 } QUIC_LISTENER_STATISTICS;
 
+typedef enum QUIC_PERFORMANCE_COUNTERS {
+    ConnectionsCreated,
```

Comment:
All out enums are all caps and prefixed with the type. So `QUIC_PERF_COUNT_CONNECTIONS_CREATED` maybe?